offchain/advance-runner/src/main.rs

```diff
@@ -2,16 +2,14 @@
 // SPDX-License-Identifier: Apache-2.0 (see LICENSE)
 
 use advance_runner::config::AdvanceRunnerConfig;
-use tracing_subscriber::filter::{EnvFilter, LevelFilter};
+use tracing::info;
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let filter = EnvFilter::builder()
-        .with_default_directive(LevelFilter::INFO.into())
-        .from_env_lossy();
-    tracing_subscriber::fmt().with_env_filter(filter).init();
-
     let config = AdvanceRunnerConfig::parse()?;
 
+    logs::configure(&config.logs_config);
+
+    info!(?config, "Starting Advance Runner");
```

Comment:
```suggestion
```

This line looks like a duplicate of the following one:

https://github.com/cartesi/rollups-node/blob/e9a9c23299a79a294d2d5ad85099a8bcdae94bfe/offchain/advance-runner/src/lib.rs#L28

Reply:
@fmoura FYI